compiler+runtime/bin/build-clang

```diff
@@ -49,7 +49,7 @@ echo "Using ${make_j} cores to build"
 
 srcdir="${PWD}"
 
-llvm_url="https://github.com/jank-lang/llvm-project.git"
+llvm_url="https://github.com/pfeodrippe/llvm-project.git"
```

Comment:
This will be reverted before merging if we merge https://github.com/jank-lang/llvm-project/pull/1 \o